src/ViewModels/ArcGisMapServerItemViewModel.js

```diff
@@ -66,6 +66,8 @@ ArcGisMapServerItemViewModel.prototype._enableInCesium = function() {
 
     var scene = this.application.cesium.scene;
 
+    this.legendUrl = this.url + '/legend';
```

Comment:
But you haven't dropped it from the init file.  It's still there, and still used.  But then when the layer is enabled, the init-specified legend gets clobbered and replaced with a different one.  If we ever put the legend on the info popup (or anywhere else you could see without enabling the layer), the user would see the init file legend before enabling it, and the server's legend afterward.

Also, I think the ability to specify a legend other than the one the server suggests is a handy bit of functionality, and it's already implemented.  I don't see any good reason to get rid of it.